scenarios/evaluate-math/evaluate-math.ipynb

```diff
@@ -0,0 +1,286 @@
+    "word stems to more accurately capture meaning and language variations. In addition to machine translation and\n",
+    "text summarization, paraphrase detection is an optimal use case for the METEOR score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.evaluation import MeteorScoreEvaluator\n",
+    "\n",
+    "meteor = MeteorScoreEvaluator(\n",
+    "    alpha=0.9,\n",
+    "    beta=3.0,\n",
+    "    gamma=0.5\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = meteor(\n",
+    "    response=\"Tokyo is the capital of Japan.\",\n",
+    "    ground_truth=\"The capital of Japan is Tokyo.\")\n",
+    "\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### RougeScoreEvaluator\n",
+    "\n",
+    "ROUGE (Recall-Oriented Understudy for Gisting Evaluation) is a set of metrics used to evaluate automatic\n",
+    "summarization and machine translation. It measures the overlap between generated text and reference summaries.\n",
+    "ROUGE focuses on recall-oriented measures to assess how well the generated text covers the reference text. Text\n",
+    "summarization and document comparison are among optimal use cases for ROUGE, particularly in scenarios where text\n",
+    "coherence and relevance are critical.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.evaluation import RougeScoreEvaluator, RougeType\n",
+    "\n",
+    "rouge = RougeScoreEvaluator(rouge_type=RougeType.ROUGE_1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = rouge(\n",
+    "    response=\"Tokyo is the capital of Japan.\",\n",
+    "    ground_truth=\"The capital of Japan is Tokyo.\")\n",
+    "\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate a Dataset using Math Evaluators\n",
+    "\n",
+    "The code below uses the Evaluate API with BLEU, GLEU, METEOR, and ROUGE evaluators to evaluate the results on a dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from azure.ai.evaluation import evaluate\n",
+    "\n",
+    "azure_ai_project = {\n",
+    "    \"subscription_id\": os.environ.get(\"AZURE_SUBSCRIPTION_ID\"),\n",
+    "    \"resource_group_name\": os.environ.get(\"AZURE_RESOURCE_GROUP\"),\n",
+    "    \"project_name\": os.environ.get(\"AZURE_PROJECT_NAME\"),\n",
+    "}\n",
+    "\n",
+    "result = evaluate(\n",
+    "    data=\"data.jsonl\",\n",
+    "    evaluators={\n",
+    "        \"bleu\": bleu,\n",
```

Comment:
isn't the keyword param `bleu_score`, `gleu_score`, `meteor_score`, `rouge_score` ?

Reply:
![image](https://github.com/user-attachments/assets/8e8704fa-cd0a-4b75-a990-d6cccb60344e)